java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -111,6 +111,7 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
         baseTransaction.zmscore(key8, new String[] {"two", "three"});
         baseTransaction.zrange(key8, new RangeByIndex(0, 1));
         baseTransaction.zrangeWithScores(key8, new RangeByIndex(0, 1));
+        baseTransaction.zrangestore(key8, key8, new RangeByIndex(0, -1));
```

Comment:
add `zrangestore` with 4 args

Reply:
No need this calls the 4 argument function in itself.

Reply:
so use it here instead for better coverage

Reply:
They get the same coverage, if you don't specify it you get even better coverage. Because the one with 3 args calls the func with 4 args.

Reply:
oh my mistake - all goes vice versa
Sorry